mindsdb/integrations/handlers/huggingface_handler/huggingface_handler.py

```diff
@@ -0,0 +1,137 @@
+import pandas as pd
+import transformers
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class HuggingFaceHandler(BaseMLEngine):
+    name = 'huggingface'
+
+    def create(self, target, args=None, **kwargs):
+
+        args['target'] = target
+
+        model_name = args['model_name']
```

Comment:
here, the value of  model_name, I think its passing by the "using block" ,  do you want to get the name after "create"?
(example: create huggingface.abc ...  , here the abc is model_name). If you want to get the one after create, I think maybe you can get it by querying of predictor_id from db.

Reply:
Generally speaking, you're right. However, I think this in particular is related to a few subtleties on HuggingFace itself. @tomhuds can expand, but we require this because one thing is what name do you store this model with in the MindsDB predictors table, but another one is the URL to the hub that you use to actually call it. The `model_name` here references the latter, and so you need both of them to get a working connection. Makes sense?

Reply:
> Generally speaking, you're right. However, I think this in particular is related to a few subtleties on HuggingFace itself. @tomhuds can expand, but we require this because one thing is what name do you store this model with in the MindsDB predictors table, but another one is the URL to the hub that you use to actually call it. The `model_name` here references the latter, and so you need both of them to get a working connection. Makes sense?

I got it, thanks ~